vlib/vweb/README.md

```diff
@@ -61,3 +61,77 @@ That means that the template automatically has access to that action's entire en
 ### Deploying vweb apps
 
 Everything, including HTML templates, is in one binary file. That's all you need to deploy.
+
+## Getting Started
+
+To start with vweb, you have to import the module `vweb` first.
+After it, it's recommended to define a struct to access `vweb.Context`.
+The web server can be started by calling `vweb.run<App>(port)`.
+A complete
+
+**Example:**
+```v ignore
+import vweb
+
+struct App {
+    vweb.Context
+}
+
+fn main() {
+	vweb.run<App>(8080)
+}
+```
+
+### Defining endpoints
+To add endpoints to your web server, you have to extend the `App` struct.
+The function names are automatically mapped to the URL path and a return 
+value of `vweb.Result` is expected.
+
+**Example:**
+```v ignore
+fn (mut app App) hello() vweb.Result {
+	return app.text('Hello')
+}
+```
+
+This endpoint can be accessed via `http://localhost:port/hello`.
+
+To create an HTTP POST endpoint, you simply add a `[post]` attribute before the function definition.
+
+**Example:**
+```v ignore
+[post]
+fn (mut app App) world() vweb.Result {
+	return app.text('World')
+}
+```
+
+Instead of using the auto-mapping, you can 
+also use an attribute
+to define a different path to the endpoint.
+
+**Example:**
+```v ignore
+[post]
+["/foo"]
+fn (mut app App) world() vweb.Result {
+	return app.text('World')
+}
+```
+
+To pass a param to an endpoint, you simply define it inside an attribute, e. g. `['/hello/:user]`.
+After it, you have to add it as a function parameter.
```

Comment:
After it is defined in the attribute, you have to add it as a function parameter.